docs/workflow/testing/libraries/testing-android.md

```diff
@@ -0,0 +1,57 @@
+# Testing Libraries on Android
+
+To build the tests and run them on Android (devices or emulators) you need the following prerequisites.
+
+- [Android NDK](https://developer.android.com/ndk/downloads)
+- [Android SDK](https://developer.android.com/studio)
+- OpenJDK
+
+(TODO: provide command-line steps to download them with dependencies)
+
+Once SDKs are downloaded, set `ANDROID_NDK_ROOT`(CLARIFY: looks like `ANDROID_NDK_HOME` should also be set) and `ANDROID_SDK_ROOT`.
```

Comment:
where do you see ANDROID_NDK_HOME being used? it shouldn't be needed

Reply:
I removed it and will bring back if I notice it's needed